benchexec/tablegenerator/react-table/src/components/SelectColumn.js

```diff
@@ -52,24 +52,35 @@ export default class SelectColumn extends React.Component {
   };
 
   renderColumns = index => {
-    return this.state.list[index].columns.map(column => (
-      <td
-        id={"td" + index + column.display_title}
-        key={"key" + index + column.display_title}
-        className={column.isVisible ? "checked" : ""}
-      >
-        <label>
-          {column.display_title}
-          <input
-            id={index + "--" + column.display_title}
-            name={index + "--" + column.display_title}
-            type="checkbox"
-            checked={column.isVisible}
-            onChange={this.handleSelecion}
-          ></input>
-        </label>
-      </td>
-    ));
+    const self = this;
```

Comment:
Is there a specific reason for having `self` instead of just using an arrow function that does not bind `this`?

Reply:
It was leading to an error, that's why I used `self`, but using an arrow function seems to be a better solution.